homeassistant/components/sensor/scrape.py

```diff
@@ -34,6 +35,7 @@
     vol.Required(CONF_RESOURCE): cv.string,
     vol.Required(CONF_SELECT): cv.string,
     vol.Optional(CONF_ATTR): cv.string,
+    vol.Optional(CONF_INDEX): cv.positive_int,
```

Comment:
Set the default here. This make the test on line 125 obsolete.

```suggestion
    vol.Optional(CONF_INDEX, default=0): cv.positive_int,
```

Reply:
Thanks for the suggestion